integrations/SnapPixel/util.js

```diff
@@ -54,6 +55,7 @@ const ecommEventPayload = (event, message) => {
     payment_info_available: get(message, "properties.payment_info_available"),
     sign_up_method: get(message, "properties.sign_up_method"),
     success: get(message, "properties.success"),
+    transaction_id: get(message, "properties.transaction_id"),
```

Comment:
```suggestion
    transaction_id: get(message, "properties.transactionId") || get(message, "properties.transaction_id"),
```
Lets support camel casing for properties root level attributes

Reply:
Done